### DISCUSSION TITLE
2023/12/10/%E5%9F%BA%E4%BA%8EGitalk%E5%92%8CGithub%E6%90%AD%E5%BB%BA%E4%B8%80%E4%B8%AA%E8%AF%84%E8%AE%BA%E5%8C%BA/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#2] [Announcements]
# 2023/12/10/%E5%9F%BA%E4%BA%8EGitalk%E5%92%8CGithub%E6%90%AD%E5%BB%BA%E4%B8%80%E4%B8%AA%E8%AF%84%E8%AE%BA%E5%8C%BA/

好久不见！在咕咕咕了好几个月后，今天回来修下评论区了！

https://blog.byteloid.one/2023/12/10/%E5%9F%BA%E4%BA%8EGitalk%E5%92%8CGithub%E6%90%AD%E5%BB%BA%E4%B8%80%E4%B8%AA%E8%AF%84%E8%AE%BA%E5%8C%BA/

**Comment by bingxin666** [comment]
### 不建议使用Gitalk了，建议使用Giscus
### 折腾了一个月都没修好Error: no access token

**Comment by Seayay** [comment]
Giscus 是好文明
